design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,147 @@
+type Mutation {
```

Comment:
It looks like there is no way to proceed to checkout, how was it supposed to be done?
My understanding is that negotiable quote should be converted to cart for proceeding. Please correct me if it works differently.